client/app/services/record-event.js

```diff
@@ -0,0 +1,25 @@
+import { debounce, extend } from 'lodash';
```

Comment:
Filename should be `recordEvent.js`. See: 
https://github.com/airbnb/javascript#naming--filename-matches-export

Reply:
Well, we don't follow this rule almost nowhere :smile:

Reply:
We follow it in all _new_ files since we adopted Airbnb's styleguide/rules. Old files followed a different rule.